code/drasil-example/Drasil/GamePhysics/TMods.hs

```diff
@@ -44,22 +46,21 @@ newtonTLNote = foldlSent [(S "Every action has an equal and opposite reaction" !
 
 -- T3 : Newton's law of universal gravitation --
 
+-- FIXME: Missing ConceptDomain!
+newtonLUGModel :: ModelKinds
```

Comment:
I wanted this one to be the prototype of sorts for the EquationalRealm since it was the original model that started this work. I think it makes sense to work with at least one model while building the EquationalRealms so that we know what to expect them to look like.